includes/parser/Parser.php

```diff
@@ -2132,13 +2132,17 @@ private function maybeMakeExternalImage( $url ) {
 		if ( $this->mOptions->getAllowExternalImages()
 			|| ( $imagesexception && $imagematch )
 		) {
-			if ( preg_match( self::EXT_IMAGE_REGEX, $url ) ) {
+			if ( preg_match( self::EXT_IMAGE_REGEX, $url )
+				 || Hooks::run( 'ParserAllowExternalImage', [ $url ] )
+			) {
 				# Image found
 				$text = Linker::makeExternalImage( $url );
 			}
 		}
 		if ( !$text && $this->mOptions->getEnableImageWhitelist()
-			&& preg_match( self::EXT_IMAGE_REGEX, $url )
+			 && ( preg_match( self::EXT_IMAGE_REGEX, $url )
+				  || Hooks::run( 'ParserAllowExternalImage', [ $url ] )
```

Comment:
Let's mark this with a comment as our change

Reply:
Do we have "standard" for that comment? eg `// Fandom`

Reply:
In app we did something like that: https://github.com/Wikia/app/blob/dev/includes/Message.php#L569-L575, with mariking changes begin and end